packages/vendure-plugin-shipmate/README.md

```diff
@@ -15,6 +15,9 @@ import { ShipmatePlugin } from '@pinelab/vendure-plugin-shipmate';
 plugins: [
   ShipmatePlugin.init({
     apiUrl: 'https://api.shipmate.co.uk/v1.2', // Or https://api-staging.shipmate.co.uk/v1.2 for the testing environment
+    shouldSendOrder: function ( ctx: RequestContext, order: Order): Promise<boolean> | boolean {
+      return order.totalQuantity < 5;
+    }
```

Comment:
This the the implementing code, so types are infered, and I don't think you can have a union as return type on this side? (`boolean | Promise<>`), not sure....
```suggestion
    shouldSendOrder: function ( ctx, order) {
      // Sample implementation that only sends orders under €5 to Shipmate
      return order.totalQuantity < 5;
    }
```